src/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -217,13 +217,14 @@ const Conversation: React.FC<ConversationProps> = props => {
       !!conversation.isLastMessageToUser ? 3000 : 0
     )
   }, [lastMessageID])
-  // -Workaround Reply render resizing race condition
 
+  // -Workaround Reply render resizing race condition
   useEffect(() => {
     if (initialMount.current) scrollToBottom()
     const rect = scrollContainer.current?.getBoundingClientRect()
     setToastBottom(window.innerHeight - (rect?.bottom ?? 0) + 30)
-  }, [scrollContainer?.current?.clientHeight])
+  }, [scrollContainerCurrentHeight])
+
```

Comment:
I'll come clean - this was to fix one `useEffect` error message, but there are still 5 other ones in this file 😬 I took the `--no-verify` approach since this wasn't related to the change I made and I'm not an effective enough React engineer at this moment to fix these quickly.

Reply:
I would say, unless you're certain about what items to go into the effect (to satisfy the linter) its best to avoid. There are circumstances where satisfying the hooks requirements can lead to infinite loops and whatnot. It's a rule that's best in place for new code. 

(Do feel free to fix other linter warnings / errors, however! The other ones are safe.)

Reply:
**suggestion:** Should we undo this change if you're unsure about it?

Reply:
hm yeah now that I look at this again, pretty sure it should be reverted. I moved something that could change into a const; I think the net result is that even if it changes we wouldn't notice and re-fire the `useEffect` hook unless we also had a rerender.

Not 100% confident in my explanation there, which probably tells us enough 😄 so walked it back